src/pin-code/types.js

```diff
@@ -49,6 +49,8 @@ export type PropsT = {
   manageFocus: boolean,
   /** An array of values respective to each pin code input. */
   values: string[],
+  /** Mask for pin. Default is no mask. Set it true then mask is ".". Also accept string input as custom mask style.  */
+  mask: boolean | string,
```

Comment:
we also need to update the `index.d.ts` for typescript defs

Reply:
OK